### ISSUE TITLE
How to make the report always uses specify path?

### ISSUE THREAD

**Issue by MelonHjy** [istanbuljs/nyc#1583]
When generating HTML reports with nyc, the directory paths for coverage reports are inconsistent. 
For example, when the coverage information includes directories /User/test/pc/common, /User/test/pc/public/src/services, and /User/test/pc/public/src/pages, the resulting report's coverage directory paths appear as common, public/src/services, and public/src/pages. 
However, when the coverage information includes only the /User/test/pc/public/src/services and /User/test/pc/public/src/pages directories, the resulting report's coverage directory paths are services and pages. 
How can I fix the paths so that the report always uses the part of the path after /User/test/pc? 
In other words, regardless of whether /User/test/pc/common is included, the coverage directory paths in the report should always be public/src/services and public/src/pages.

**Comment by MelonHjy** [comment]
<img width="719" alt="image" src="https://github.com/user-attachments/assets/2d218cca-3b2b-4156-9a6a-7bb9d485cbd8">
